BetterErrors.use_pry! if defined?(BetterErrors)
